examples/plugins/config-plugin/main_linux.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package main
+
+import (
+	"log"
+	"path/filepath"
+
+	"github.com/sylabs/singularity/internal/pkg/cgroups"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+	pluginapi "github.com/sylabs/singularity/pkg/plugin"
+	singularity "github.com/sylabs/singularity/pkg/runtime/engines/singularity/config"
+)
+
+// Plugin is the only variable which a plugin MUST export.
+// This symbol is accessed by the plugin framework to initialize the plugin
+var Plugin = pluginapi.Plugin{
+	Manifest: pluginapi.Manifest{
+		Name:        "sylabs.io/config-plugin",
+		Author:      "Sylabs Team",
+		Version:     "0.0.1",
```

Comment:
Again, as per [this semver recommendation](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase), should be `0.1.0`.